### DISCUSSION TITLE
Bind to trash in connector

### DISCUSSION THREAD

**Discussion by nmpetkov** [Studio-42/elFinder#3327] [Q&A]
I know it is possible to bind when delete a file,

'rm' => 'bindcallbackfunction'

But it is useless when you activate trash volume - then file is removed and placed in trash folder.

Is it possible to bind to both rm and trash, something like:

'rm trash' => 'bindcallbackfunction'

**Comment by nao-pon** [comment]
@nmpetkov The command that trash actually uses is paste, so listen to the paste command.
